src/interfaces.ts

```diff
@@ -326,10 +326,14 @@ export interface WatchedCronItems {
   release: () => void;
 }
 
+export type CronItemOptions = {
+  [key in keyof ParsedCronItemOptions]?: ParsedCronItemOptions[key];
+};
```

Comment:
Does this maintain the documentation for these fields in TypeScript hover/etc?

Reply:
![Screenshot 2024-12-04 at 5 21 17 PM](https://github.com/user-attachments/assets/71ee0541-4455-49ff-957b-e7f1a8cd3bce)

I believe so